servatrice/migrations/servatrice_0024_to_0025.sql

```diff
@@ -0,0 +1,6 @@
+-- Servatrice db migration from version 24 to version 25
+
+ALTER TABLE `cockatrice_uptime` ADD COLUMN `mods_count` int(11) NOT NULL DEFAULT 0;
+ALTER TABLE `cockatrice_uptime` ADD COLUMN `mods_list` BLOB;
```

Comment:
If you are storing text and not binary data, `TEXT` is a better choice.

Reply:
Never knew the diff, thanks!